src/main/resources/static/js/global-variables.js

```diff
@@ -1,6 +1,6 @@
 var router = new Navigo(null, true, '#');
 var frontendServiceUrl = $('#frontendServiceUrl').text();
```

Comment:
as reference in other places with this also not have "" inside ?

Reply:
Will fix in next commit.

Reply:
Done.